### PR TITLE
doc: add cdn doc + migration to cdn

### PR DESCRIPTION
**Pull request by sguiheux**
1. Description
1. Related issues
1. About tests
1. Mentions

@ovh/cds

### CODE REVIEW

**Comment by ovh-cds** [comment]
CDS Report build-all-cds#14403.0 ✘  
* Build
  * Build and Package All ✔ 
  * Build Debpacker ✘  
  * Build OS-Ansible-Inventory ✔ 
  * Build SMTPMock ✔ 
  * Docs -   
  * Lint UI Translate ✔ 
  * Test CLI ✔ 
  * Test Engine ✔ 
  * Test SDK ✔ 
  * Test SDK Rust ✔ 
  * Test UI ✔

File: docs/content/docs/components/cdn/_index.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+title: "CDN"
+weight: 3
+---
+
+<b style="color: red">⚠ Do not activate CDN log processing in production yet. It's in active development.
+Be sure that config flag 'enableLogProcessing' is set to false</b>
+
+## What's CDN
+
+CDN is a service dedicated to receice and retrieve logs. It replaces the old system that stored logs in the CDS database
```
Comment (yesnault) [inline docs/content/docs/components/cdn/_index.md:11]:
```suggestion
CDN is a service dedicated to receive and retrieve logs. 
```

File: docs/content/docs/components/cdn/_index.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+A CDN instance is started with scope Project and Run using token.
+
+CDN is linked to 2 types of storages:
+
+* Buffer Unit: to store step log temporarily during the execution
```
Comment (yesnault) [inline docs/content/docs/components/cdn/_index.md:17]:
```suggestion
* Buffer Unit: to store step log temporarily during the execution of a job
```

**Review by yesnault — CHANGES_REQUESTED** [review]

**Comment by ovh-cds** [comment]
CDS Report build-all-cds#14408.0 ✘  
* Build
  * Build and Package All ✔ 
  * Build Debpacker ✔ 
  * Build OS-Ansible-Inventory ✔ 
  * Build SMTPMock ✔ 
  * Docs -   
  * Lint UI Translate ✔ 
  * Test CLI ✔ 
  * Test Engine ✔ 
  * Test SDK ✔ 
  * Test SDK Rust ✔ 
  * Test UI ✔ 
* Package and Analyze
  * Docker ✔ 
  * Sonar ✔ 
* Integration
  * Docker Compose Tests ✔ 
  * Standalone Tests ✘  
  * Standalone tests 1 -   
  * Standalone tests 2 -   
  * Standalone tests 3 -

**Review by yesnault — APPROVED** [review]

**Review by richardlt — APPROVED** [review]

**Pull request merged**